README.rst

```diff
@@ -101,6 +101,15 @@ We’d love to get contributions from you! Once you are up and running,
 take a look at the `Contribution Document`_ to see how to get your
 changes merged in.
 
+Log Insight and vROps make outbound HTTP connections to your shim, with a JSON body containing alerts details.
+The shim is responsible for parsing the body and identifying key information relevant to its operation.
+The shim makes one or more upstream connections to some other service using a native library or raw HTTP, performing a needed task.
+
+At completion, the shim reports back an overall success or failure using the HTTP status code; :code:`200` is treated as success while all other results are treated as a failure.
```

Comment:
I disagree -- assume you had native webhook support between LI and vRO, is the expectation that vRO return a 200 OK? The shim is a shim not a translator. We may in the short-term before a translation, but this is due to a bug in webhook originator, not a problem in the shim.